docs/checks.md

```diff
@@ -97,28 +97,27 @@ commit.
 
 This test has tiered scoring. Each tier must be fully satisfied to achieve points at the next tier. For example, if you fulfill the Tier 3 checks but do not fulfill all the Tier 2 checks, you will not receive any points for Tier 3.
 
-Note: If Scorecard is run without an administrative access token, the requirements that specify “For administrators” are ignored.
+Note: If Scorecard is run without an administrative access token, the requirements that specify “For administrators” can be safely ignored, and scores will be determined as if all such requirements have been met.
 
 Tier 1 Requirements (3/10 points):
   - Prevent force push
   - Prevent branch deletion
   - For administrators: Do not allow bypassing the above settings
 
 Tier 2 Requirements (6/10 points):
-  - Required reviewers >=1
-  - For administrators: Last push review
-  - For administrators: Strict status checks (require branches to be up-to-date before merging)
-
-Tier 3 Requirements (8/10 points):
-  - Status checks defined
+  - Require at least 1 reviewer for approval before merging
+  - For administrators: Require branch to be up to date before merging
+  - For administrators: Require approval of the most recent reviewable push
 
+Tier 3 Requirements (7/10 points):
```

Comment:
I think this actually gives 8/10.
I tested it using Scorecard 4.10.4 on project flutter/packages

Reply:
You're right. As per my tests, the points by Tier are a bit weird. In my understanding, it should be what I wrote in the documentation, but for some reason it's not.

Reply:
Documenting here the problems with the score:
- Using non-admin token, when you reach tier 3 you have 8/10 and should be 7/10.
- Using non-admin token, when you reach tier 3 you have 8/10. When enabling "Require review from code owners" you still get an 8/10.
- Using admin token, when you reach tier 1 you have 4/10 and should be 3/10.
- Using admin token, when you reach tier 2 and enable "Require at least 1 reviewer for approval before merging " you have 5/10. When enabling "For administrators: Require branch to be up to date before merging" you still get a 5/10.
- Using admin token, when you reach tier 3 you have 8/10 and should be 7/10.
- Using admin token, when you reach tier 3 you have 8/10. When enabling "Require at least 2 reviewers for approval before merging" you still get an 8/10.
- Using admin token, when you reach tier 3 you have 8/10. When enabling "Require review from code owners" you still get an 8/10.

Reply:
What I can change right now so the documentation makes more sense is:
- Tier 1 (3/10 or 4/10 points)
- Tier 3 (8/10 points)
But really this scoring should be fixed.

Reply:
I think some clarity from @laurentsimon or other maintainers could be useful here.

Reply:
> For me, the score should be count 1 point for each setting but when using non-admin token if you reached all the tier settings you could do you get the max points for that tier.

@laurentsimon Do you agree with this scoring?

Reply:
> Using non-admin token, when you reach tier 3 you have 8/10 and should be 7/10.

That's because by adding enabling more setting in the tier, it does not update?

I think what happened is that we defined some levels, but over time more settings were introduced, so enabling certain settings does not always have the intended effect on the score.. because it's not granular enough.

I'n not against fixing the scores. Sometimes I wonder if we should have score from 1-100 :)

Reply:
> That's because by adding enabling more setting in the tier, it does not update the score?

That's because the user enables 3 settings for tier 1, 3 settings for tier 2, and 1 setting for tier 3. I believe the user should get 7/10 at this point, but for some reason gets 8/10. So, it does update the score, but 1 more point than necessary.

Reply:
https://github.com/ossf/scorecard/issues/2796

Reply:
We can further discuss the score problem in the new issue. For the documentation to make sense for now I will add the following changes:
> What I can change right now so the documentation makes more sense is:
> - Tier 1 (3/10 or 4/10 points)
> - Tier 3 (8/10 points)

Reply:
Please reopen if you disagree

Reply:
I'd suggest "(3/10 with admin token, 4/10 without)" or something.